packages/cli-platform-ios/src/tools/installPods.ts

```diff
@@ -110,14 +110,18 @@ async function installCocoaPods(loader: Ora) {
   }
 }
 
-async function installPods(loader?: Ora, options?: PodInstallOptions) {
+async function installPods(
+  loader?: Ora,
+  options?: PodInstallOptions,
+  iosFolderPath?: string,
```

Comment:
Can we change the order so that options object is the last? This is a convention that config objects are the last argument
```suggestion
  loader?: Ora,
  iosFolderPath?: string,
  options?: PodInstallOptions,
```

Reply:
done ✅